pages/integrations/github.md.erb

```diff
@@ -85,6 +85,12 @@ Optionally, select one or more of the following:
 
 If you want to run builds only on pull requests, set the _Branch Filter Pattern_ in the pipeline to a branch name that will never occur (such as "this-branch-will-never-occur"). Pull request builds ignore the _Branch Filter Pattern_, and all pushes to other branches that don't match the pattern are ignored.
 
+## Running builds on tags
+
+To run builds for commit tags, edit the GitHub settings for your Buildkite pipeline and choose the _Build Tags_ checkbox.
```

Comment:
I've not heard "commit tags" before, only "git tags". If git tags is the accepted term we should be consistent, but if some folks use commit tags there is value in using both to make it more findable. 

We could settle on 

> git tags (also known as commit tags) 

if  that is common